docs/source/notebooks/pavics_thredds.ipynb

```diff
@@ -52,13 +54,13 @@
        "<xarray.Dataset>\n",
        "Dimensions:    (bnds: 2, lat: 160, lon: 320, plev: 23, time: 120)\n",
        "Coordinates:\n",
-       "  * time       (time) datetime64[ns] 1991-01-16T12:00:00 ... 2000-12-16T12:00:00\n",
```

Comment:
You reverted my previous output fix https://github.com/Ouranosinc/pavics-sdi/pull/108/commits/95147960b3fc532b6ea4d5a44fa6381402d44c5d and this breaks Jenkins again, see http://jenkins.ouranos.ca/job/PAVICS-e2e-workflow-tests/job/master/85/console

I forgot to tell you that you can use our Jenkins to test your own pavics-sdi branch since when requesting a build, you can specify any pavics-sdi branch.  No need to launch your own notebook from the PAVICS-e2e-workflow-tests anymore, easier now :)